config/zones/RO.yaml

```diff
@@ -89,13 +89,19 @@ emissionFactors:
       - datetime: '2022-01-01'
         source: EU-ETS, ENTSO-E 2022
         value: 1085.6
+      - datetime: '2023-01-01'
+        source: EU-ETS, ENTSO-E 2023
+        value: 1224.26
```

Comment:
Increased production from several of the highest emitting coal power plants is driving up the average here.